src/app/content/components/ContentExcerpt.tsx

```diff
@@ -46,17 +49,20 @@ const ContentExcerpt = styled((props: Props) => {
   )(props.content), [props.content, excerptSource.url]);
 
   return <div
+    ref={forwardedRef}
     dangerouslySetInnerHTML={{ __html: fixedContent }}
     className={`content-excerpt ${className}`}
     {...excerptProps}
   />;
-})`
+};
+
+export default styled(React.forwardRef<HTMLElement, Props>(
+  (props, ref) => <ContentExcerpt {...props} forwardedRef={ref} />)
```

Comment:
would this be simpler if you put the forwardRef call around the ContentExcerpt definition?

Reply:
I don't think so